Assets/Scripts/Controllers/MainMenuController.cs

```diff
@@ -62,10 +74,14 @@ public void Start()
 
         // Dev Console
         GameObject devConsole = (GameObject)Instantiate(Resources.Load("UI/Console/DevConsole"));
-        devConsole.name = "DevConsole";
-        devConsole.transform.SetParent(canvas.transform, false);
-        devConsole.transform.SetAsLastSibling();
-        devConsole.SetActive(false);
-        DeveloperConsole.DevConsole.Close();
+
+        if (devConsole != null)
```

Comment:
They are called safety or guard checks.  We essentially 100% know it should exist but if it doesn't then who cares no need to do stuff with it.  One way it could be null is if someone wanted to delete the prefab of it, then this would catch it and make sure the game is still playable (for example we may wanted to delete the dev Console for 'commercial releases' though unlikely).  There are a ton of reasons but really in the end it doesn't effect gameplay nor speed and it just adds a little bit of safety (to make us all feel a little better :P)